src/smt/process_assertions.cpp

```diff
@@ -256,31 +251,28 @@ bool ProcessAssertions::apply(Assertions& as)
   }
 
   // rephrasing normal inputs as sygus problems
-  if (!d_slv.isInternalSubsolver())
+  if (options::sygusInference())
   {
-    if (options::sygusInference())
-    {
-      d_passes["sygus-infer"]->apply(&assertions);
-    }
-    else if (options::sygusRewSynthInput())
-    {
-      // do candidate rewrite rule synthesis
-      d_passes["synth-rr"]->apply(&assertions);
-    }
+    d_passes["sygus-infer"]->apply(&assertions);
+  }
+  else if (options::sygusRewSynthInput())
```

Comment:
```suggestion
  else if (options().quantifiers.sygusRewSynthInput)
```
Maybe also convert other calls to `options::foo()` when modifying this file anyway...

Reply:
I'll do this separately.